man/kernel-install.xml

```diff
@@ -264,6 +262,16 @@
             <para>Some other layout not understood natively by <command>kernel-install</command>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>auto</term>
+          <listitem>
+            <para>Pick layout automatically. If the kernel is a UKI set layout to <option>uki</option>. If
+            not default to <option>bls</option> if
+            <filename>$BOOT/<replaceable>ENTRY-TOKEN</replaceable></filename> exists, or
```

Comment:
```suggestion
            not default to <option>bls</option> if <filename>$BOOT/loader/entries.srel</filename> with content <literal>type1</literal> or
            <filename>$BOOT/<replaceable>ENTRY-TOKEN</replaceable></filename> exists, and
```
since the `entries.srel` file is specifically mentioned in the spec.